rel/console.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
+set -euo pipefail
+
+# This script connects to one of the running ECS tasks and opens an
+# interactive rails shell attached directly to the running system.
+# Idea stolen from https://github.com/thewca/wca-live/blob/main/rel/console.sh
+# You will need to have AWS CLI + Systems Manager Addon installed and credentials configured
+# in a profile called 'wca'
+
+# Colors for output
+COLOR_DEFAULT='\033[0m'
+COLOR_RED='\033[0;31m'
+COLOR_GREEN='\033[0;32m'
+
+export AWS_DEFAULT_PROFILE=wca
+
+usage() {
+  printf "${COLOR_DEFAULT}Usage: $0 -e <environment>\n"
+  printf " -e <environment>  Specify the environment (production or staging)"
+  exit 1
+}
+
+# Check if AWS CLI is installed
+printf "${COLOR_DEFAULT}AWS CLI | "
+command -v aws >/dev/null 2>&1 && status="$?" || status="$?"
+if [[ "${status}" = 0 ]]; then
+  cliVersion=$(aws --version)
+  printf "${COLOR_GREEN}OK ${COLOR_DEFAULT}(${cliVersion})\n"
+else
+  printf "${COLOR_RED}Missing\n"
+  exit 1
+fi
+
+# Check whether the Session Manager plugin exists
+printf "${COLOR_DEFAULT}Session Manager Plugin | "
+command -v session-manager-plugin >/dev/null 2>&1 && status="$?" || status="$?"
+if [[ "${status}" = 0 ]]; then
+  smpVersion=$(session-manager-plugin --version)
+  printf "${COLOR_GREEN}OK ${COLOR_DEFAULT}(${smpVersion})\n"
+else
+  printf "${COLOR_RED}Missing\n"
+  exit 1;
+fi
+
+# Check if you are connected to the AWS WCA Account
+printf "${COLOR_DEFAULT}aws wca credentials | "
+account=$(aws iam list-account-aliases --query "AccountAliases[0]" --output text )
+if [[ "${account}" = "thewca" ]]; then
```

Comment:
```suggestion
if [[ "${account}" == "thewca" ]]; then
```

With `[[` you can use `==`, which is probably more intuitive. Alternatively, you can do `[ "${account}" = "thewca" ]`, which is sh-compatible. Either way is fine though!